oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHWDiskStore.java

```diff
@@ -270,6 +300,27 @@ private static void computeDiskStats(LinuxHWDiskStore store, String devstat) {
         store.transferTime = devstatArray[UdevStat.ACTIVE_MS.ordinal()];
     }
 
+    private static String getPartitionNameForDmDevice(String vgName, String lvName) {
+        return new StringBuilder().append(DEV_LOCATION).append("/").append(vgName).append("/").append(lvName).toString();
+    }
+
+    private static String getMountPointOfDmDevice(String vgName, String lvName) {
+        return new StringBuilder().append(DEV_MAPPER).append("/").append(vgName).append("-").append(lvName).toString();
```

Comment:
Similar to above comment, use single quotes for single characters and possibly include the first slash in the DEV_MAPPER string